packages/http/src/__tests__/http-prism-instance.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { IPrism } from '@stoplight/prism-core';
+import { IHttpOperation } from '@stoplight/types/http-spec';
+import { omit } from 'lodash';
+import { relative, resolve } from 'path';
+import { createInstance, IHttpConfig, IHttpRequest, IHttpResponse } from '../';
```

Comment:
Completely not important, but most of the places import from `@stoplight/...`, not `'../'`

Reply:
Yes because it's actually a bit different ;)